core/lyrics/sources.go

```diff
@@ -0,0 +1,41 @@
+package lyrics
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+
+	"github.com/navidrome/navidrome/model"
+)
+
+func fromEmbedded(mf *model.MediaFile) (model.LyricList, error) {
+	if mf.Lyrics != "" {
+		return mf.StructuredLyrics()
+	}
+
+	return nil, nil
+}
+
+func fromExternalFile(mf *model.MediaFile, suffix string) (model.LyricList, error) {
+	basePath := mf.AbsolutePath()
+	ext := filepath.Ext(basePath)
```

Comment:
I think this should be path.Ext(), as paths are stored in the DB with `/`, never with local `PathSeparator`, IIRC.